package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "react-streamfield",
+  "version": "0.6.10",
+  "author": "NoriPyt <contact@noripyt.com> (https://noripyt.com)",
+  "description": "Powerful field for inserting multiple blocks with nesting.",
+  "keywords": [
+    "react",
+    "react-component",
+    "field"
+  ],
+  "homepage": "https://github.com/noripyt/react-streamfield",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/noripyt/react-streamfield"
+  },
+  "bugs": {
+    "url": "https://github.com/noripyt/react-streamfield/issues"
+  },
+  "license": "BSD",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "main": "src/index.js",
+  "devDependencies": {
+    "@babel/cli": "7.1.5",
+    "@babel/core": "7.1.6",
+    "@babel/plugin-proposal-class-properties": "7.1.0",
+    "@babel/plugin-proposal-decorators": "7.1.6",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/preset-env": "7.1.6",
+    "@babel/preset-react": "7.0.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-loader": "8.0.4",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.21",
+    "classnames": "2.2.6",
+    "css-loader": "1.0.1",
+    "mini-css-extract-plugin": "0.4.5",
+    "node-sass": "4.10.0",
+    "prop-types": "15.6.2",
+    "react": "16.6.3",
+    "react-animate-height": "2.0.7",
+    "react-beautiful-dnd": "10.0.2",
+    "react-dom": "16.6.3",
+    "react-redux": "5.1.1",
+    "redux": "4.0.1",
+    "redux-thunk": "2.3.0",
+    "sass-loader": "7.1.0",
+    "style-loader": "0.23.1",
+    "uglifyjs-webpack-plugin": "2.0.1",
+    "uuid": "3.3.2",
+    "webpack": "4.26.1",
+    "webpack-cli": "3.1.2",
+    "babel-jest": "23.6.0",
+    "jest": "23.6.0"
+  },
+  "babel": {
+    "presets": [
+      "@babel/preset-env",
+      "@babel/preset-react"
+    ],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      [
+        "@babel/plugin-proposal-decorators",
+        {
+          "legacy": true
+        }
+      ],
+      "@babel/plugin-proposal-object-rest-spread",
+      [
+        "transform-react-remove-prop-types",
+        {
+          "mode": "remove",
+          "removeImport": true,
```

Comment:
Considering how important data validation would be for a project like this, I think it might be worth shipping the proptypes in the packaged code, wrapped in dev env checks, so they don't affect the production build size, but can be used in dev mode of projects consuming this package. Something like:

```
[
      "transform-react-remove-prop-types",
      {
        "mode": "unsafe-wrap",
        "ignoreFilenames": ["node_modules"]
      }
    ]
```